setup.py

```diff
@@ -66,9 +74,129 @@
 }
 requires['full'] = sum(list(requires.values()), [])
 
+
+class BuildManPage(Command):
```

Comment:
I would appreciate if these classes also moved into a module, eg setup_support.py. it would be great to keep setup.py primarily declarative